aiidalab_widgets_base/process.py

```diff
@@ -30,13 +30,28 @@ def get_running_calcs(process):
 class SubmitButtonWidget(VBox):
     """Submit button class that creates submit button jupyter widget."""
 
-    def __init__(self, process, input_dictionary_function, description="Submit"):
-        """Submit Button
-        :process: work chain to run
-        :param_funtion: the function that generates input parameters dictionary
+    def __init__(self,
+                 process,
+                 input_dictionary_function,
+                 description="Submit",
+                 disable_after_submit=True,
+                 append_output=False):
+        """Submit Button widget.
+
+        process (Process): Process to submit.
+
+        input_dictionary_function (func): Function that generates input parameters dictionary.
+
+        description (str): Description written on the submission button.
+
+        disable_after_submit (bool): Whether to disable the button after the process was submitted.
+
+        append_output (bool): Whether to clean the widget output before every submission.
         """
 
         self.process = None
+        self.disable_after_submit = disable_after_submit
+        self.append_output = append_output
         self._process_class = process
```

Comment:
It's confusing that the constructor argument is called `process`, but the corresponding class attribute is called `_process_class` and not even accessible.

Reply:
you are right, the constructor argument should be named `process_class` for consistency